### ISSUE TITLE
AccessToken class incompatible with firebase/php-jwt 6.x

### ISSUE THREAD

**Issue by marickvantuil** [googleapis/google-auth-library-php#399]
#### Environment details

  - OS:
  - PHP version: 8.1
  - Package name and version: google/auth v1.21.0

#### Steps to reproduce

1. Verify a legitimate token with google/auth v1.21.0 and firebase/php-jwt 6.x (v6.1.2)

#### Code example

```php
(new AccessToken())->verify(
    $token,
    [
        'audience' => $audience,
        'throwException' => true,
    ]
);
```

The AccessToken class decodes the token the '5.x way' instead of the '6.x way' (which now uses a Key class). I noticed [in the PR that adds firebase/php-jwt v6](https://github.com/googleapis/google-auth-library-php/pull/391) the OAuth2 class was changed, but the AccessToken class was skipped. Or is that because the AccessToken class is experimental?

**Comment by bshaffer** [comment]
Thanks for bringing this up. I'll try to get a fix in quickly!

**Comment by bshaffer** [comment]
fixed in #402
